deployment/ansible/group_vars/development

```diff
@@ -18,6 +18,8 @@ graphite_host: "{{ services_ip }}"
 statsite_host: "{{ services_ip }}"
```

Comment:
I added this to https://github.com/WikiWatershed/model-my-watershed/pull/621 instead, since it is more likely to be included in the near future than this P/R.